bin/ci

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash --login
```

Comment:
Ah, it didn't work: https://circleci.com/gh/codeclimate/ruby-test-reporter/133 and I undid it. I think the loss in portability is a thing, but not a huge thing, because we don't change CI services that often... WDYT?